qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummaryReport.java

```diff
@@ -107,7 +107,7 @@ public void toXml(Element parent1) {
 	 * 
 	 * @return next row in file
 	 */
-	public void parseRecord(FastqRecord record) {
+	public void parseRecord(FastqRecord record) throws Exception {
```

Comment:
are any exceptions thrown from this method? If not, please remove, otherwise be more specific in terms of the type of exception